txstratum/cli.py

```diff
@@ -28,6 +28,7 @@ def create_parser() -> ArgumentParser:
     parser.add_argument('--tx-timeout', help='Tx mining timeout (seconds)', type=int, default=None)
     parser.add_argument('--fix-invalid-timestamp', action='store_true', help='Fix invalid timestamp to current time')
     parser.add_argument('--prometheus', help='Path to export metrics for Prometheus', type=str, default=None)
+    parser.add_argument('--prometheus-port', help='Enables exporting metrics in a http port', type=int, default=9090)
```

Comment:
If we have this default value, then how can this be optional? If I don't pass the `--prometheus-port` param, does it use 9090?

Reply:
That's a good point